market/demos/excel-connector/meta.json

```diff
@@ -0,0 +1,73 @@
+{
+	"id": "excel-connector-demo",
+	"version": "9.3",
+	"name": "Excel Connector Demo",
```

Comment:
No 'Demo' here. I think it's a left-over from the HtmlDialogDemos where we copied form